recipes/perl-text-unidecode/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "perl-text-unidecode" %}
+{% set version = "1.30" %}
+{% set sha256 = "6c24f14ddc1d20e26161c207b73ca184eed2ef57f08b5fb2ee196e6e2e88b1c6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/S/SB/SBURKE/Text-Unidecode-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  run_exports:
+    weak:
+      - {{ name }} ={{ version }}
+
+requirements:
+  host:
+    - perl =>5.32
+    - make  # [not win]
+    - m2-make  # [win]
+
+  run:
+    - perl =>5.32
+
+test:
+  imports:
+    - Text::Unidecode
+
+
+
+about:
+  home: http://metacpan.org/pod/Text-Unidecode
+  license: perl_5
```

Comment:
can you maybe change the license to something like that https://github.com/conda-forge/perl-uri-feedstock/blob/main/recipe/meta.yaml#L41

Important is that its correct ;) and its a SPDX identifier. https://spdx.dev/

Reply:
Hi Bjorn, 

I used the same licence as the example Perl package you sent me. I checked the package licence and they said they are using the same Perl licence, I searched for a Perl licence and found this

 Perl is an Open Source software, licensed under its Artistic License, or the GNU General Public License (GPL)
 
 As long as there is no different licence in the tool and they say they use a Perl licence I will use this form.